oak_cameras:
  "1944301081DB714800": #deviceId for oak0
    name: oak0
    params:
      camera:
        i_enable_imu: false
        i_enable_ir: false
        i_floodlight_brightness: 0
        i_laser_dot_brightness: 100
        i_nn_type: none
        i_pipeline_type: RGBD
        i_usb_speed: SUPER_PLUS
      rgb:
        i_board_socket_id: 0
        i_fps: 30.0
        i_height: 720
        i_interleaved: false
        i_max_q_size: 10
        i_preview_size: 250
        i_enable_preview: false
        i_low_bandwidth: true
        i_keep_preview_aspect_ratio: true
        i_publish_topic: true
        i_resolution: '720P'
        i_width: 1280
      # stereo:
      #   i_depth_preset: ROBOTICS

      #   # Limit usable range to where the sensor is reliable
      #   i_enable_threshold_filter: true
      #   i_threshold_filter_min_range: 400    # 0.4 m
      #   i_threshold_filter_max_range: 3000   # 3.0 m (or 4.0 if it looks good)

      #   # Make OAK more conservative about what counts as valid depth
      #   i_stereo_conf_threshold: 100         # higher -> discard more noisy pixels

      #   # Temporal filter to smooth flicker
      #   i_enable_temporal_filter: true
      #   i_temporal_filter_alpha: 0.4
      #   i_temporal_filter_persistency: VALID_2_IN_LAST_4

      #   # Spatial (edge-preserving) smoothing
      #   i_enable_spatial_filter: true
      #   i_spatial_filter_hole_filling_radius: 2
      #   i_spatial_filter_alpha: 0.5
      use_sim_time: false
  "19443010B1627E4800": #deviceId for oak1
    name: oak2
    params:
      camera:
        i_enable_imu: false
        i_enable_ir: false
        i_floodlight_brightness: 0
        i_laser_dot_brightness: 0
        i_nn_type: none
        i_pipeline_type: RGBD
        i_usb_speed: SUPER_PLUS
      rgb:
        i_board_socket_id: 0
        i_fps: 30.0
        i_height: 720
        i_interleaved: false
        i_max_q_size: 10
        i_preview_size: 250
        i_enable_preview: false
        i_low_bandwidth: true
        i_keep_preview_aspect_ratio: true
        i_publish_topic: true
        i_resolution: '720P'
        i_width: 1280
      use_sim_time: false
usb_cameras:
  "2450ZB1354S8": # USB camera0 serial number
    name: camera0
    params:
      image_width: 640
      image_height: 480
      pixel_format: mjpeg2rgb 
      auto_white_balance: false
      auto_exposure: false
      auto_focus: true
      framerate: 24.0